docs/walkthrough-1.7.md

```diff
@@ -88,14 +88,14 @@ Notice that the `status` field has been set to reflect that the broker server's
 catalog of service offerings has been successfully added to our cluster's
 service catalog.
 
-# Step 3 - Viewing `ServiceClass`es
+# Step 3 - Viewing `ClusterServiceClass`es
 
-The controller created a `ServiceClass` for each service that the UPS broker
-provides. We can view the `ServiceClass` resources available in the cluster by
+The controller created a `ClusterServiceClass` for each service that the UPS broker
+provides. We can view the `ClusterServiceClass` resources available in the cluster by
 executing:
 
 ```console
-$ kubectl get serviceclasses -o=custom-columns=NAME:.metadata.name,EXTERNAL\ NAME:.spec.externalName
+$ kubectl get clusterserviceclasses -o=custom-columns=NAME:.metadata.name,EXTERNAL\ NAME:.spec.externalName
```

Comment:
Oh my God hahah, I never noticed this line here before. So simple and elegant. 🤣